src/pvrclient-nextpvr.cpp

```diff
@@ -357,13 +357,17 @@ bool cPVRClientNextPVR::IsUp()
         }
       }
     }
-    else if (m_nowPlaying == Transcoding)
+    else if (m_nowPlaying != NotPlaying)
```

Comment:
Is `NotPlaying` a constant/enum?

Reply:
> 
> 
> Is `NotPlaying` a constant/enum?

It's an enum 

```
enum eNowPlaying
{
  NotPlaying = 0,
  TV = 1,
  Radio = 2,
  Recording = 3,
  Transcoding
};
```

Reply:
Cool, I think I’m just used to enums being CAPS_AND_UNDERSCORES so it looked a little odd to me.